src/sentry/api/endpoints/organization_teams.py

```diff
@@ -80,9 +80,11 @@ def get(self, request: Request, organization) -> Response:
         if request.auth and hasattr(request.auth, "project"):
             return Response(status=403)
 
-        queryset = Team.objects.filter(
-            organization=organization, status=TeamStatus.VISIBLE
-        ).order_by("slug")
+        queryset = (
```

Comment:
@getsentry/app-backend is it preferred to use `prefetch_related_objects` in the serializer `get_attrs` instead?

Reply:
Yeah, agree that it's generally better to put this in `get_attrs` in the serializer. Otherwise any consumers of the serializer need to remember to use `select_related`

Reply:
Going to handle it as @k-fish suggested, using this PR as a demo and making a second one to move it to get_attrs.